algopy_testing/src/algopy/op.py

```diff
@@ -0,0 +1,68 @@
+import hashlib
+
+import nacl.exceptions
+import nacl.signing
+from Cryptodome.Hash import keccak
+
+from algopy import Bytes
+
+
+def sha256(a: Bytes | bytes, /) -> Bytes:
+    if not isinstance(a, bytes | Bytes):
```

Comment:
We can move `algopy.primitives._utils` to `algopy.utils` and then use `as_bytes` here to get a `bytes` that has been checked for type and length

Reply:
Done